src/components/feedback-widgets/feedback-widget-custom-element.shadow.css

```
@@ -0,0 +1,459 @@
+:host {
+	& slot[name="title"] {
+		/* Layout */
+		margin-block-end: 5--step;
+
+		/* Text */
+		font-size: 20--rpx;
+		font-weight: 500;
+
+		@media (width < 1024px) {
+			display: none;
+		}
+	}
+
+	& .widget_content {
+		/* Layout */
+		position: relative;
+
+		& h3 {
+			/* Layout */
+			margin-block-end: 5--step;
+
+			/* Text */
+			font-size: 20--rpx;
+			font-weight: 500;
+		}
+
+		& p {
+			/* Text */
+			font-size: 16--rpx;
+		}
+
+		& button {
+			/* Layout */
+			padding-block: 1--step;
+			padding-inline: 4--step;
+
+			/* Text */
+			font-size: 16--rpx;
+			line-height: 24--rpx;
+
+			/* Appearance */
+			background-color: #ffffff;
+
+			/* Appearance */
+			border: 1px solid #005a8f;
+
+			/* Appearance */
+			border-radius: 3px;
+			cursor: pointer;
+
+			&.widget_vote-button {
```

```suggestion
			&.widget_vote-button {
			    display: flex;
```
The feedback vote buttons aren't perfectly square, if you display them flex then the svg stops taking in the line height or whatever nonsense its doing.

---

All set!